Source/JavaScriptCore/runtime/VM.cpp

```diff
@@ -1885,4 +1886,15 @@ void VM::DrainMicrotaskDelayScope::decrement()
     }
 }
 
+#if ENABLE(WEBASSEMBLY)
+
+Wasm::DebugState* VM::debugState()
+{
+    if (!m_debugState)
+        m_debugState = makeUnique<Wasm::DebugState>();
```

Comment:
This might be a silly question, but since this is Wasm is it possible for multiple worker threads to access this at once?